### PR TITLE
Invibes Bid Adapter : change referrer formatting

### PR DESCRIPTION
**Pull request by mihaisandu07**
## Type of change
- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [x] Other

## Description of change
Removed cutoff of url parameters from location param sent to AdServer.

### CODE REVIEW

**Review by patmmccann — APPROVED** [review]

**Comment by mihaisandu07** [comment]
@patmmccann I've added a unit test and all unit tests are now passing. The only error is triggered by failed test on gothamadsBidAdapter_spec on which I don't have control

**Comment by patmmccann** [comment]
Tests failed to run, circleci says 0 of 0 complete

File: modules/invibesBidAdapter.js [hunk 1]
```diff
@@ -372,7 +372,7 @@ function generateRandomId() {
 }
 
 function getDocumentLocation(topWin) {
-  return topWin.location.href.substring(0, 300).split(/[?#]/)[0];
+  return topWin.location.href.substring(0, 300);
```
Comment (patmmccann) [inline modules/invibesBidAdapter.js:375]: Why not use site.page on the request object

File: test/spec/modules/invibesBidAdapter_spec.js [hunk 2]
```diff
@@ -323,6 +323,13 @@ describe('invibesBidAdapter:', function () {
       expect(parsedData.height).to.exist;
     });
 
+    it('has location not cut off', function () {
+      const request = spec.buildRequests(bidRequests, {auctionStart: Date.now()});
+      const parsedData = request.data;
+      expect(parsedData.location).to.exist;
```
Comment (patmmccann) [inline test/spec/modules/invibesBidAdapter_spec.js:329]: Write out an actual input and output

File: modules/invibesBidAdapter.js (hunk 1 above)
Comment (patmmccann) [reply modules/invibesBidAdapter.js:375]: prebid team and community have put a lot of work into making sure https://github.com/prebid/Prebid.js/blob/master/src/refererDetection.js gives the best possible results. Publishers and buyers often expect consistent determinations of site.page across integrations.

Comment (rcheptanariu) [reply modules/invibesBidAdapter.js:375]: Hello @patmmccann ! I understand the concern but you are requesting a  change in the Invibes adapter that is not part of the scope of this pull request. From my knowledge this is not standard prebid policy.

This pull request however was for a bugfix on one of our integrations, the change was minor (removal of split function, NOT access to window object) and we needed to implement it as fast as possible (it has been dragging on for weeks at this point)

We will of course implement this if required as a SEPARATE issue, with a different pull request and we require the appropriate time for it.

Thank you for understanding

Comment (patmmccann) [reply modules/invibesBidAdapter.js:375]: This is not out of scope; you are accessing the page url incorrectly. Your pr revises how to gather the pageurl. It remains incorrect

Comment (mihaisandu07) [reply modules/invibesBidAdapter.js]: Hey @patmmccann, we have made the requested changes

Comment (mihaisandu07) [reply modules/invibesBidAdapter.js]: Hey @patmmccann , an update please?

**Comment by mihaisandu07** [comment]
@patmmccann Can you please review? We made the requested changes more than 2 weeks ago.

**Review by patmmccann — CHANGES_REQUESTED** [review]
please delete the getconfig and make sure you pass tests

We've had a holiday in the United States, not much got reviewed last week,

File: modules/invibesBidAdapter.js [hunk 3]
```diff
@@ -371,8 +372,13 @@ function generateRandomId() {
   return (Math.round(Math.random() * 1e12)).toString(36).substring(0, 10);
 }
 
-function getDocumentLocation(topWin) {
-  return topWin.location.href.substring(0, 300).split(/[?#]/)[0];
+function getDocumentLocation(bidderRequest) {
+  var site = getConfig('site') || {};
+
+  if (!site.page) {
+    site.page = bidderRequest.refererInfo.page;
```
[the remark this answers is not in the document]
Comment (mihaisandu07) [reply modules/invibesBidAdapter.js]: hey @patmmccann , I've simplified the mode to get the page url and removed the config. 

Regarding the unit tests, all of our unit tests are passsing, the errors are on gothamadsBidAdapter_spec.js and e_volutionBidAdapter_spec.js which are NOT our adapters so they are our of our control. 

Please let me know if everything else is ok.

[the remark this answers is not in the document]
Comment (patmmccann) [reply modules/invibesBidAdapter.js]: Yes; looking into test failure, hopefully should have an answer quickly

**Comment by ChrisHuie** [comment]
@mihaisandu07 can you pull in recent commits on master into this pr. Should fix the errors.

**Comment by mihaisandu07** [comment]
@ChrisHuie thanks, I've synced the MASTER and everything looks ok now.

**Pull request merged**